app/controllers/candidate_interface/gcse_details_controller.rb

```diff
@@ -1,15 +1,83 @@
 module CandidateInterface
   class GcseDetailsController < CandidateInterfaceController
```

Comment:
Considering we have `#edit_type`, `#update_type`, `#edit_details` and `#update_details`, it would be an idea create a namespace for our `GCSE` controllers. E.g.

- `Gcse::TypeController`
- `Gcse::DetailsController`
- and maybe a `Gcse::ReviewController`

With both having their own `#edit` and `#update` actions.